docs/pages/docs/handbook/building-your-app.mdx

```diff
@@ -0,0 +1,86 @@
+import { Tabs, Tab } from '../../../components/Tabs'
+import Callout from '../../../components/Callout'
+
+# Building your App
+
+Unless your monorepo is only used for [publishing packages to NPM](/docs/handbook/publishing-packages/basics), it will likely contain at least one application. Co-ordinating your app's builds with Turborepo can lead to some extraordinary gains in speed.
```

Comment:
```suggestion
Unless your monorepo is only used for [publishing packages to NPM](/docs/handbook/publishing-packages/basics), it will likely contain at least one application. Coordinating your app and package builds with Turborepo can lead to some extraordinary gains in speed.
```

Reply:
This section is about building your app, not package.

Reply:
"Coordinating" made it sound like you're teaching users about how Turborepo can coordinate dependencies between workspaces. Two apps probably don't need to "coordinate" anything, they just build 🤷🏾